job_service/api/request_models.py

```diff
@@ -7,9 +7,9 @@
 
 
 class CommandEnum(str, Enum):
-    ADD_DATASET = 'ADD_DATA'
-    DELETE_DATASET = 'REMOVE'
-    CHANGE_DATASET = 'CHANGE_DATA'
+    ADD_OR_CHANGE_DATA = 'ADD_OR_CHANGE_DATA'
+    REMOVE_DATASET = 'REMOVE'
+    CHANGE_DATASET = 'IMPORT'
```

Comment:
Remove?

Reply:
is REMOVE not the correct command terminology for setting a dataset to pending_delete?